build.sbt

```diff
@@ -35,6 +35,12 @@ lazy val higherKindCore = project settings (
   libraryDependencies ++= Seq(catsCore, catsFree, catsTagless),
 )
 
+lazy val kernel = project in file("kernel") dependsOn (opticsCore, higherKindCore) settings (
```

Comment:
What I feel to to best here is to make `core` dependent on `kernel`, moving desired type classes instead of copying them

Reply:
Done